api/pkg/models/models.go

```diff
@@ -5,9 +5,18 @@ import (
 	"github.com/google/martian/har"
 	"github.com/up9inc/mizu/shared"
 	"github.com/up9inc/mizu/tap"
+	"mizuserver/pkg/utils"
 	"time"
 )
 
+type IEntry interface {
```

Comment:
https://golang.org/doc/effective_go#interface-names
Call it `DataUnmarshaller`.